plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SequencePropertiesEditionPartForm.java

```diff
@@ -1326,6 +1325,18 @@ public void focusGained(FocusEvent e) {
 		return parent;
 	}
 	
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, dynamicReferenceKey);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```

Comment:
Remove commented line.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804